packages/contracts/contracts/L1/StateRootOracle.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.9.0;
+
+// This should be an UPGRADABLE CONTRACT.
+// We will eventually need to add support for concurrent
+// state root proposals, but until we have a full dispute
+// game spec that we are happy with, it is better to KEEP IT SIMPLE.
+contract StateRootOracle {
```

Comment:
@cfromknecht for the purpose of this PR, would it be possible to add just the interface for this contract, or the minimum viable amount of contract you need to implement the go-code?

Reply:
No problem, I stripped down the contract to get to an MVP 👍